yafti/screen/title.py

```diff
@@ -37,10 +41,57 @@ class Config(YaftiScreenConfig):
         title: str
         description: str
         icon: Optional[str] = None
+        links: List[dict[str, str]] = None
```

Comment:
I think this needs to be wrapped in Optional? Not sure though if Optional is inferred with assignment of a default value. Either way, with the move to a plugin config the second variable is a dict

```suggestion
        links: List[dict[str, dict]] = None
```